js/util/is_char_in_unicode_block.js

```diff
@@ -1,280 +1,281 @@
+    // 'Byzantine Musical Symbols': (char: number) => char >= 0x1D000 && char <= 0x1D0FF,
+    // 'Musical Symbols': (char: number) => char >= 0x1D100 && char <= 0x1D1FF,
+    // 'Ancient Greek Musical Notation': (char: number) => char >= 0x1D200 && char <= 0x1D24F,
+    // 'Tai Xuan Jing Symbols': (char: number) => char >= 0x1D300 && char <= 0x1D35F,
+    // 'Counting Rod Numerals': (char: number) => char >= 0x1D360 && char <= 0x1D37F,
+    // 'Mathematical Alphanumeric Symbols': (char: number) => char >= 0x1D400 && char <= 0x1D7FF,
+    // 'Sutton SignWriting': (char: number) => char >= 0x1D800 && char <= 0x1DAAF,
+    // 'Glagolitic Supplement': (char: number) => char >= 0x1E000 && char <= 0x1E02F,
+    // 'Mende Kikakui': (char: number) => char >= 0x1E800 && char <= 0x1E8DF,
+    // 'Adlam': (char: number) => char >= 0x1E900 && char <= 0x1E95F,
+    // 'Arabic Mathematical Alphabetic Symbols': (char: number) => char >= 0x1EE00 && char <= 0x1EEFF,
+    // 'Mahjong Tiles': (char: number) => char >= 0x1F000 && char <= 0x1F02F,
+    // 'Domino Tiles': (char: number) => char >= 0x1F030 && char <= 0x1F09F,
+    // 'Playing Cards': (char: number) => char >= 0x1F0A0 && char <= 0x1F0FF,
+    // 'Enclosed Alphanumeric Supplement': (char: number) => char >= 0x1F100 && char <= 0x1F1FF,
+    // 'Enclosed Ideographic Supplement': (char: number) => char >= 0x1F200 && char <= 0x1F2FF,
+    // 'Miscellaneous Symbols and Pictographs': (char: number) => char >= 0x1F300 && char <= 0x1F5FF,
+    // 'Emoticons': (char: number) => char >= 0x1F600 && char <= 0x1F64F,
+    // 'Ornamental Dingbats': (char: number) => char >= 0x1F650 && char <= 0x1F67F,
+    // 'Transport and Map Symbols': (char: number) => char >= 0x1F680 && char <= 0x1F6FF,
+    // 'Alchemical Symbols': (char: number) => char >= 0x1F700 && char <= 0x1F77F,
+    // 'Geometric Shapes Extended': (char: number) => char >= 0x1F780 && char <= 0x1F7FF,
+    // 'Supplemental Arrows-C': (char: number) => char >= 0x1F800 && char <= 0x1F8FF,
+    // 'Supplemental Symbols and Pictographs': (char: number) => char >= 0x1F900 && char <= 0x1F9FF,
+    // 'CJK Unified Ideographs Extension B': (char: number) => char >= 0x20000 && char <= 0x2A6DF,
+    // 'CJK Unified Ideographs Extension C': (char: number) => char >= 0x2A700 && char <= 0x2B73F,
+    // 'CJK Unified Ideographs Extension D': (char: number) => char >= 0x2B740 && char <= 0x2B81F,
+    // 'CJK Unified Ideographs Extension E': (char: number) => char >= 0x2B820 && char <= 0x2CEAF,
+    // 'CJK Compatibility Ideographs Supplement': (char: number) => char >= 0x2F800 && char <= 0x2FA1F,
+    // 'Tags': (char: number) => char >= 0xE0000 && char <= 0xE007F,
+    // 'Variation Selectors Supplement': (char: number) => char >= 0xE0100 && char <= 0xE01EF,
+    // 'Supplementary Private Use Area-A': (char: number) => char >= 0xF0000 && char <= 0xFFFFF,
+    // 'Supplementary Private Use Area-B': (char: number) => char >= 0x100000 && char <= 0x10FFFF,
```

Comment:
Could we instead define the exported object type (`type UnicodeBlockLookup = {[string]: (number) => number}` etc.) and avoid setting types on every line?